src/utils/constants.ts

```diff
@@ -8,4 +8,11 @@ export const NETWORKS = {
 
 export const TCR_DECIMALS = 18;
 export const TEST_TOKEN_DECIMALS = 18;
-export const USDC_DECIMALS = 6;
\ No newline at end of file
+export const USDC_DECIMALS = 6;
+
+export const TEST_QUOTE_TOKEN = {
+    name: 'USDC',
+    symbol: 'USDC',
```

Comment:
don't think it matters and in fact probably will break icons but this is actually `TUSDC`